python/kokkos/__init__.py.in

```diff
@@ -0,0 +1,74 @@
+#!@Python3_EXECUTABLE@
+# MIT License
```

Comment:
Does this make sense put directly into Kokkos? Or should we make a separate project that can pull in Kokkos/KokkosTools/KokkosKernels/etc.